runtime/index.d.ts

```diff
@@ -12,16 +19,64 @@ declare module '@sapper/app' {
 }
 
 declare module '@sapper/server' {
-	import { Handler, Req, Res } from '@sapper/internal/manifest-server';
+	import { IncomingMessage, ServerResponse } from 'http';
+	import { TLSSocket } from 'tls';
 
 	export type Ignore = string | RegExp | ((uri: string) => boolean) | Ignore[];
 
+	/**
+	 * The request object passed to middleware and server-side routes. 
+	 * These fields are common to both Polka and Express, but you are free to 
+	 * instead use the typings that come with the server you use.
+	 */
+	export interface Req extends IncomingMessage {
```

Comment:
should we call this `Request` instead of `Req`?

Reply:
Actually, if we just call them `Request` and `Response` do we need to specify the types here? See the comment in https://github.com/sveltejs/sapper/pull/1603#issuecomment-709429933

Reply:
Yeah, these namings are very confusing. `Request` and `Response` are the types used by `fetch`. They are similar but unrelated to the request / response types used by `http.Server` which are called `IncomingMessage` and `ServerResponse`.

Polka and Express, in turn, each extend the `IncomingMessage` in different ways, but with a core set of attributes that are identical in both. Those are the ones we have in `Req`.

That said, I'd definitely welcome a renaming. How about `SapperRequest` / `SapperResponse`?

Reply:
Ah, I see. Ok. `SapperRequest` and `SapperResponse` sound fine to me

Reply:
Ok, I renamed them.